pages/common/speedtest.md

```diff
@@ -0,0 +1,33 @@
+# speedtest
+
+> Official command line interface for testing internet bandwidth using https://speedtest.net.
+> Note: some platforms link `speedtest` to `speedtest-cli`. If some of the examples in this page don't work, try `tldr speedtest-cli`.
```

Comment:
Perhaps this will read better?
```suggestion
> Note: some platforms link `speedtest` to `speedtest-cli`. If some of the examples in this page don't work, see `speedtest-cli`.
```